### DISCUSSION TITLE
Global menu

### DISCUSSION THREAD

**Discussion by darkoverlordofdata** [elementary/triage#578] [Ideas]
I understand the pov on global menu, it's been talked about on various message boards, and thank you all for designing your applications so well without the menus!

However I spend over 90% of my time wrangling menus on the various third party apps I use - libreOffice, Ardour, Guitarix, Krita, VSCode and their support programs such as QJackCtl.

I much prefer the global menu orientaion for all these other windows, so I'm not constantly randomized looking for the menu, it the ux just wears me out.  

You've got so many other features I love. I've looked at elementaryOS time and again over the years and it just get's better and better, but I always end up dropping it for this reason.

**Comment by danirabbit** [comment]
Sorry, there’s just no way to do this on a technical level. There is no method for applications to export a menubar outside of the application on Linux. And as far as I know, nobody is developing one

Many modern apps no longer use menubar-based designs and the design language of our platform and other popular Linux platforms don’t use menubars

this is something that will almost certainly never happen for both design and technical reasons
